src/components/events-page/events-feed.jsx

```diff
@@ -22,6 +23,8 @@ const FlexRow = styled.div`
     display: flex;
 `;
 
+const AddEventButton = FilterMenuLink.extend``;
```

Comment:
semantically it should be a button, so you can use `withComponent`

Reply:
Done!